src/evaluator.rs

```diff
@@ -19,24 +23,14 @@ impl<'a> Evaluator {
         }
     }
 
-    /// Evaluates statements at global scope until one of them returns.
+    /// Evaluate the ast's statements at global scope until one of them returns
```

Comment:
Nit: Prefer `Evalautes` to `Evaluate`, the second could be misinterpreted as an instruction for the caller.